libr/include/r_util/r_strbuf.h

```diff
@@ -9,21 +9,25 @@ typedef struct {
 	int len;
 	char *ptr;
 	int ptrlen;
-	char buf[64];
+	char buf[32];
 } RStrBuf;
 
 #define R_STRBUF_SAFEGET(sb) (r_strbuf_get (sb) ? r_strbuf_get (sb) : "")
 R_API RStrBuf *r_strbuf_new(const char *s);
 R_API bool r_strbuf_set(RStrBuf *sb, const char *s);
+R_API bool r_strbuf_setbin(RStrBuf *sb, const ut8 *s, int len);
```

Comment:
Do we really need those? Imho they can be done by calling strbuf-set. Also I’m not sure they fit well with the rest of the api because the are supposed to work with string-like objects. What if the data passed does not contain a null And append or other functions are called?

Reply:
rbuffer is the alternative to strbuf but its much more heavy. i think that it doesnt hurts too much if we just use rstrbuf with binary too, so we can also save utf16 strings , pascal, etc

Reply:
Why do you need strbuf at all?
1. What you are trying to store is not a string at all
2. It may not be null terminated, which is an assumption in many places of strbuf 
3. You are not using strbuf_set, append, or any of the other functions, so why don’t you just make it a ut8*? It seems to me you just need to replace the entire buffer every time so imho a free +malloc+memcpy is enough